### PR TITLE
[#158907283] Replace the old spid-testenv-identityserver with the new spid-testenv2

### PR DESCRIPTION
**Pull request by lussoluca**
The autologin user is configured as an environment variable. We can set that variable in the CI environment to perform tests.
At the moment we are blocked by https://github.com/italia/spid-testenv2/pull/118

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Remember to fix the PR title by removing WIP wording when ready
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#158907283](https://www.pivotaltracker.com/story/show/158907283): Come test di integrazione dell'app, quando devo autenticarmi, voglio poter usare un IdP finto che mi faccia accedere automaticamente, in modo da non dover inserire username e password

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/container.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ container.register({
 const SAML_CALLBACK_URL =
   process.env.SAML_CALLBACK_URL ||
   "http://italia-backend/assertionConsumerService";
-const SAML_ISSUER = process.env.SAML_ISSUER || "http://italia-backend";
+const SAML_ISSUER = process.env.SAML_ISSUER || "http://italiabackend.it";
```
Comment (gunzip) [inline src/container.ts]: why this one ?

Comment (lussoluca) [reply src/container.ts]: because in the new testenv2 the issuer has to be a valid URL

Comment (gunzip) [reply src/container.ts]: is there any reason on why this differs from the default saml issuer (https://spid.agid.gov.it/cd) ?
I mean, cant we just put https://spid.agid.gov.it/cd into the sp_metatadata.xml of the test spid IDP ?

File: src/container.ts [hunk 2]
```diff
@@ -221,7 +224,7 @@ container.register({
 function createSimpleRedisClient(): redis.RedisClient {
   const redisUrl = process.env.REDIS_URL || "redis://redis";
   log.info("Creating SIMPLE redis client", { url: redisUrl });
-  return redis.createClient();
+  return redis.createClient(redisUrl);
```
Comment (gunzip) [inline src/container.ts]: may you please open a new PR for this one ? (or a pivotal bug)

**Comment by gunzip** [comment]
In other words, this PR should add just one more entry to the spidStrategy.ts with the test IDP info.

**Pull request merged**